### PR TITLE
Adding cluster ops api for anthos

### PR DESCRIPTION
**Pull request by jainsmit**
**What this PR does / why we need it**:
These APIs need to perform Anthos cluster operations 

**Which issue(s) this PR fixes** (optional)
Closes # PTX-21976

**Special notes for your reviewer**:

### CODE REVIEW

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:38]: You don't need to specify name of the object you returning, you can just do `(*v1beta1.Cluster, error)` like you are doing in the function below. Please lets be consistent

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:47]: Let's return these things here `(*v1beta1.Cluster, error)`

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:23]: You don't need to specify name of the object you returning, you can just do (*v1beta1.Cluster, error) like you are doing in the function below. Please lets be consistent

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:34]: Let's return these things here (*v1beta1.Cluster, error)

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:42]: Change this to `cluster` like you are using in the function below, result is not very intuitive for this var, let's be consistent
NOTE: Also should it be `:=`, since this is first time you defining this var?

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:27]: Change this to `cluster` like you are using in the function below, result is not very intuitive for this var, let's be consistent

**Review by nikolaypopov — CHANGES_REQUESTED** [review]
Please don't mix 2 styles in 3 functions

File: k8s/anthos/clusters.go
Comment (jainsmit) [reply k8s/anthos/clusters.go:23]: Incorporated !!

File: k8s/anthos/clusters.go
Comment (jainsmit) [reply k8s/anthos/clusters.go:27]: Incorporated !!

File: k8s/anthos/clusters.go
Comment (jainsmit) [reply k8s/anthos/clusters.go]: Incorporated !!

File: k8s/anthos/clusters.go
Comment (jainsmit) [reply k8s/anthos/clusters.go]: Incorporated !!

File: k8s/anthos/clusters.go
Comment (nikolaypopov) [inline k8s/anthos/clusters.go:51]: Can we change this function to something like `GetClusterStatus()` because it seems that what it returns and please have same comment on tops of the function like `// GetClusterStatus gets cluster and returns its status`, also add same description on tops on line 19

File: k8s/anthos/clusters.go
Comment (jainsmit) [reply k8s/anthos/clusters.go]: Done

File: k8s/anthos/clusters.go
Comment (jainsmit) [reply k8s/anthos/clusters.go]: Incorporated !!

**Review by nikolaypopov — APPROVED** [review]
LGTM as long as you tested it in the actual place you are going to vendor in and use these

**Comment by jainsmit** [comment]
I will create a new PR as lot of updates went after PR approval..

**Pull request closed without merging**
